src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.css';
+
+const list = document.querySelector('.list');
+
+function displayLists(toDo) {
+  const text = `<li class="item">
+                       <i class="uil uil-square circle-icon" job="complete"></i>
+                       <p class="text" > ${toDo}<p>
+                       <i class="uil uil-trash-alt" job="delete"></i>
+               </li>`;
+
+  const position = 'beforeend';
+  list.insertAdjacentHTML(position, text);
+}
+const arr = [];
```

Comment:
* I suggest you make an array of some simple to do tasks (array of objects) as it is a project requirement. Each task object should contain three keys eg [description [string], completed [bool], index: [number].